src/Kushy/SymbolLoader.cs

```diff
@@ -19,16 +19,27 @@ namespace Kushy
     /// </summary>
     public class SymbolLoader
     {
-        private readonly string _defaultConnection;
+        private readonly KustoConnectionStringBuilder _defaultConnection;
         private readonly string _defaultClusterName;
         private readonly HashSet<string> _ignoreClusterNames = new HashSet<string>();
         private readonly Dictionary<string, HashSet<string>> _badDatabaseNameMap = new Dictionary<string, HashSet<string>>();
 
         /// <summary>
-        /// Creates a new <see cref="SymbolLoader"/> instance.
+        /// Creates a new <see cref="SymbolLoader"/> instance. recommended method: SymbolLoader(KustoConnectionStringBuilder clusterConnection)
         /// </summary>
         /// <param name="clusterConnection">The cluster connection string.</param>
-        public SymbolLoader(string clusterConnection)
+        public SymbolLoader(string strClusterConnection)
```

Comment:
You can just make this constructor call the other constructor directly passing it the newly constructed connection string builder. This avoids having two separate constructors initializing the same fields.

Reply:
Chained the constructors.